### ISSUE TITLE
"Not interested" back to top of the list. 

### ISSUE THREAD

**Issue by tiru92** [yuliskov/SmartTubeNext#1429]
Im probably the minority here but ive developed a muscle memory for holding the button and immediately pressing it again so i can choose "not interested" for a particular video.

Thats how it used to be until the recent update, the "play" option is the topmost in the list now.

If i wanted to play a video i wouldnt be holding down the button, i would just press it once.

Small QoL change, hopefully it gets the attention.

**Comment by Ketec** [comment]
Yes. This is annoying. I regularly have to hit this because of the horrible manipulative algorithm but now I'm instead hitting play. And then I also have to go to history to delete it or my recommendations become a nightmare.

**Comment by tiru92** [comment]
> Yes. This is annoying. I regularly have to hit this because of the horrible manipulative algorithm but now I'm instead hitting play. And then I also have to go to history to delete it or my recommendations become a nightmare.
> 
> 

Ughh bro i feel you. Though it seems not many feel the same as us. Otherwise the thread would have gotten more replies.

Suprised not many are annoyed by that change.

**Comment by Jahfry** [comment]
I wrote #823 for the awhile back. It would address a lot of UX issues and prevent complaints that always get filed when the order changes in a new patch.

**Comment by costinmx** [comment]
> Suprised not many are annoyed by that change.

Maybe this is the reason:
You do have the option to remove 'Play' from Context Menu settings, then 'Not Interested' it's back in first position.
But I also support the option to rearange items. However, I bet this is not an easy task.

At times I wonder if developer has some sort of telemetrics about how this menus are activated and used. IMHO it would help in development.

> Thats how it used to be until the recent update, now the "play" option is the topmost in the list.
> 
> If i wanted to play a video i wouldnt be holding down the button to open up the options list and then click "play", i would just press it once on the thumbnail.

**Comment by kariudo** [comment]
This should be as simple as moving this line to the top of the array:
https://github.com/yuliskov/SmartTubeNext/blame/1d274aa874cffaf2ad0287569bc7f716c0a29315/common/src/main/java/com/liskovsoft/smartyoutubetv2/common/app/presenters/settings/GeneralSettingsPresenter.java#L114
